wandb/sdk/internal/internal_api.py

```diff
@@ -1940,18 +1944,78 @@ def upload_urls(
             project (str): The project to download
             files (list or dict): The filenames to upload
             run (str, optional): The run to upload to
-            entity (str, optional): The entity to scope this project to.  Defaults to wandb models
-            description (str, optional): description
+            entity (str, optional): The entity to scope this project to.
```

Comment:
should you put the description back? as long as we have it in the legacy one?

Reply:
yes, we should!!! just added back :)